woocommerce/woocommerce
# Add subdivisions of Laos

### Changes proposed in this Pull Request:

Based on https://github.com/unicode-org/cldr/blob/release-35-1/common/subdivisions/en.xml#L2690

@claudiosanches the only thing that I'm not sure is that we are using the subdivisions code instead of the name when displaying the full address in the order confirmation page. I'm not sure if the codes used in the CLDR standard makes sense to the users. Am I missing something?

cc @Dezueka

Closes #24762.

### How to test the changes in this Pull Request:

1. Check that the new subdivisions are displayed when selecting Laos as the country during checkout.

### Changelog entry

> i18n: add subdivisions of Laos
